### PR TITLE
remove duplicate execute_ddl, @impl true, table prefixes

### PR DESCRIPTION
**Pull request by ruslandoga**
closes https://github.com/elixir-sqlite/ecto_sqlite3/issues/102

### CODE REVIEW

File: test/ecto/adapters/sqlite3/connection_test.exs [hunk 1]
```diff
@@ -2567,34 +2562,26 @@ defmodule Ecto.Adapters.SQLite3.ConnectionTest do
...
-             """
-             CREATE INDEX "posts$main" ON "foo"."posts" (lower(permalink))\
-             """
-           ]
+    assert_raise ArgumentError, "SQLite3 does not support table prefixes", fn ->
+      execute_ddl(create)
+    end
   end
 
   test "create index with comment" do
```
Comment (ruslandoga) [inline test/ecto/adapters/sqlite3/connection_test.exs:2577]: this test is about comments but "raises" due to prefixes

File: test/ecto/adapters/sqlite3/connection_test.exs [hunk 2]
```diff
@@ -2567,32 +2562,25 @@ defmodule Ecto.Adapters.SQLite3.ConnectionTest do
...
-    create =
-      {:create,
-       index(:posts, [:category_id, :permalink], prefix: :foo, comment: "comment")}
+    create = {:create, index(:posts, [:category_id, :permalink], comment: "comment")}
 
     assert execute_ddl(create) == [
              """
              CREATE INDEX "posts_category_id_permalink_index" \
-             ON "foo"."posts" ("category_id", "permalink")\
+             ON "posts" ("category_id", "permalink")\
```
Comment (ruslandoga) [inline test/ecto/adapters/sqlite3/connection_test.exs:2583]: Since the test is about comments, I just removed the prefix option here.

Comment (warmwaffles) [reply test/ecto/adapters/sqlite3/connection_test.exs:2583]: That's good

**Review by warmwaffles — COMMENTED** [review]
Less code to maintain. I'll do another pass at this when it is ready.

**Review by warmwaffles — APPROVED** [review]
@ruslandoga can you add an entry to the changelog as well about this change? We have an unreleased section with some breaking changes that I'll roll this into a 1.0 release as well.

**Pull request merged**